l5kit/l5kit/visualization/visualiser/common.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, NamedTuple
+
+import numpy as np
+
+
+class LaneVisualisation(NamedTuple):
+    xs: np.ndarray
+    ys: np.ndarray
+    color: str
+
+
+class CWVisualisation(NamedTuple):
+    xs: np.ndarray
+    ys: np.ndarray
+    color: str
+
+
+class AgentVisualisation(NamedTuple):
+    xs: np.ndarray
+    ys: np.ndarray
+    color: str
+    track_id: int
+    type: str
```

Comment:
`type()` is already a reserved function, I would consider changing the name here.